packages/app/src/components/VisualizationOptions/Options/CheckboxBaseOption.js

```diff
@@ -3,23 +3,34 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import Checkbox from '@material-ui/core/Checkbox';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
+import { withStyles } from '@material-ui/core/styles';
 import { sGetUiOptions } from '../../../reducers/ui';
 import { acSetUiOptions } from '../../../actions/ui';
 
-export const CheckboxBaseOption = ({ option, value, onChange }) => (
+const styles = {
+    formControlLabelRoot: {},
+    checkBoxRoot: {},
+};
+
+export const CheckboxBaseOption = ({ option, value, onChange, classes }) => (
     <FormControlLabel
         control={
             <Checkbox
+                classes={{
+                    root: classes.checkBoxRoot,
+                }}
```

Comment:
When overriding the root class in Mui you can usually use `className={classes.checkBoxRoot}` instead of explicitly specifying `classes.root`